java/ql/src/semmle/code/java/dataflow/ExternalFlow.qll

```diff
@@ -403,27 +406,25 @@ predicate sinkModel(
 /** Holds if a summary model exists for the given parameters. */
 predicate summaryModel(
   string namespace, string type, boolean subtypes, string name, string signature, string ext,
-  string input, string output, string kind
+  string input, string output, string kind, string row
```

Comment:
I'd rather not rely on exposing `row`. If we at some point supplement this with data from `external` predicates, i.e. actual csv files, then the `row` data won't be available for free.

Reply:
OK, I understand what you *don't* want, but not what you *do*.


As currently designed, the user supplies a CSV file they want tests generated for, so of course I want to share the parser, and introduce `SummarizedCallableExternal.propagatesFlowForRow` to give that relationship.

Do you want a different design where they don't give a CSV file as input?